base_classes/NXtransformations.nxdl.xml

```diff
@@ -325,7 +325,8 @@
 		<attribute name="depends_on" type="NX_CHAR">
 			<doc>
 				Points to the path to a field defining the axis on which this
-				depends or the string "." when at the end of the chain.
+				depends or the string ".". It can also point to an instance of
```

Comment:
I guess we need to make the wording more precise my suggestion:
Points to the path of a field that defines the axis on which the here defined instance of NXtransformations depends or the string ".".

I suggest write out in plain language with one sentence in which case only the string should be "."

Reply:
I guess if you use ".", then you are pointing to the Mcstas system, right?

Reply:
The current (official NIAC NeXus documentation of NXtransformation states for NXtransformations.
"The NeXus default McSTAS coordinate frame is assumed, but additional useful coordinate axes may be defined by using axes for which no transformation type has been specified."
This means that traditionally effectively a set of instances of NXtransformation can be used to define a coordinate system (indeed explicitly by specifying that cs' axes)
The proposal of an own base class NXcoordinate_system possibly to take in some cases for this last role and usage of NXtransformations makes the discussion unfortunately more complicated.
NXcoordinate_system(_set) gives a suggestion how and when McStas remains used and valid.
One case discussed is when no NXcoordinate_system instance is defined in an appdef.
In this case indeed I agree with you "." means you have reached the end of the chain
See again current official NeXus def for NXtransformation AXISNAME: "Points to the path to a field defining the axis on which this depends or the string “.”." i.e. "." means the chain has ended (aka is no more dependency and thus no more translation operation to perform in addition to the already possibly defined ones.

So maybe leave it for now then it retains at the base class level its original/traditional meaning.